### ISSUE TITLE
Remove deprecated content from homepage

### ISSUE THREAD

**Issue by brianking** [mozilla/feedthefox#219]
Scrap section from the Hub home page:
- Firefox OS for Android
- Porting

**Comment by MarciaZikan** [comment]
@ppapadeas any updates?

**Comment by MarciaZikan** [comment]
@ppapadeas  Please, scrap from the Hub : 
Firefox OS for Android developer preview - I deleted this page on the flat pages but now appears page not found 404 [ Marcia ]
Gaia Development, same error 404 appears 
Add-ons , same error 404 appears
Dashboard

**Comment by comzeradd** [comment]
Besides "Firefox OS for Android developer preview" and "Gaia Development" do we also remove "Add-ons" and the Dashboard?

**Comment by comzeradd** [comment]
Also do we completely remove the Dogfood menu?

**Comment by MarciaZikan** [comment]
@comzeradd yes, please remove "Add-ons" and the Dashboard

**Comment by MarciaZikan** [comment]
@comzeradd on the Fooxfooding menu only remove the Dashboard. Please, keep the rest.

**Comment by comzeradd** [comment]
I just opened a [PR](https://github.com/mozilla/feedthefox/pull/225) that fixes most of the re-branding.

On the current homepage, besides the major Call-to-action, we have two minor contribution ones. One about Porting and one about B2GDroid. Since we are removing B2GDroid I removed it from homepage too, but I added one about the Smart TV contribution.

Is that ok? If we don't want to point people to either Porting or Smart TVs then we need to figure out what's the content of the homepage besides the About text.

/cc @brianking
